recipes/flasgger/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "flasgger" %}
+{% set version = "0.8.0" %}
+{% set md5 = "185f95e37f3d8cfc90f4eb80be0f8f72" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - jsonschema
+    - pyyaml
+    - flask
+    - mistune
+    - six
+
+
+test:
+  imports:
+    - flasgger
+
+about:
+  home: https://github.com/rochacbruno/flasgger
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Easy Swagger UI for your Flask API'
+
+extra:
+  recipe-maintainers:
+    - lvoliveira
```

Comment:
Please add mu and @igortg to this list as well. 😁

Reply:
Done